### PR TITLE
[stable/mariadb] Update to mariadb non-root image

### PR DESCRIPTION
**Pull request by tompizmor**
This PR adapts the chart to the latest MariaDB image (`10.1.28-r2`) that has been migrated to a non-root approach.

### CODE REVIEW

File: stable/mariadb/templates/deployment.yaml [hunk 1]
```diff
@@ -63,7 +71,7 @@ spec:
           mountPath: /bitnami/mariadb/conf/my_custom.cnf
           subPath: my.cnf
         - name: data
-          mountPath: /bitnami/mariadb
+          mountPath: /bitnami
```
Comment (sameersbn) [inline stable/mariadb/templates/deployment.yaml]: Changing the mountPath would break chart upgrades. If we revert this change will it be possible to perform clean chart upgrades? Ideally we should allow existing chart deployments to be upgraded whenever possible.

Comment (tompizmor) [reply stable/mariadb/templates/deployment.yaml]: You're right! For the upgrade to work, the mount path has to be `/bitnami/mariadb` for the container and the initContainer.

**Comment by sameersbn** [comment]
@tompizmor The stable/mariadb chart has undergone a couple of updates, we need to rebase this PR and bump the version accordingly.

**Comment by tompizmor** [comment]
done @sameersbn, thanks for the notice!

**Review by sameersbn — APPROVED** [review]
